server/server.ts

```diff
@@ -1,22 +1,21 @@
 import express from 'express';
 import path from 'path';
 import pg from 'pg';
+import db from "./models/index"
+
 
 const app = express();
 const port = 8080;
-
+ 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-app.get('/v1/hello', (req: express.Request, res: express.Response) => {
+app.get("/test", (req: express.Request, res: express.Response) => {
```

Comment:
```suggestion
app.get('/v1/hello', (req: express.Request, res: express.Response) => {
```

Lets keep minimal changes